### DISCUSSION TITLE
net_http_TransportRoundTrip, error: parameter are nil

### DISCUSSION THREAD

**Discussion by xiaozaiyuji** [apache/skywalking#11171] [Q&A]
I use skywalking go agent 0.1.0
The data can send by grpc success.But the agent service always print this error log.Why this happened.
skywalking-go 2023/08/01 16:29:31 execute interceptor before invoke error, instrument name: http, interceptor name: ClientInterceptor, function ID: net_http_TransportRoundTrip, error: parameter are nil

**Comment by calexfeng** [comment]
Hi, have you solved this problem then?

**Comment by gujincun** [comment]
Hi, have you solved this problem then?
